roles/tftp-server/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 netboot_image: https://deb.debian.org/debian/dists/stretch/main/installer-amd64/current/images/netboot/netboot.tar.gz
-late_command_url: https://anonscm.debian.org/cgit/debconf-video/ansible.git/plain/setup_ansible.sh
+# late_command_url: https://anonscm.debian.org/cgit/debconf-video/ansible.git/plain/setup_ansible.sh
+late_command_url: http://{{ inventory_hostname }}/lc/late.sh
```

Comment:
I find tftp harder to do R&D

wget is so much easier to use and read later than the curl command needed for tftp.